root/etc/cron.d/nextcloud

```diff
@@ -0,0 +1 @@
+*/15 * * * *   apache /opt/rh/rh-php71/root/usr/bin/php -f /usr/share/nextcloud/cron.php
```

Comment:
maybe also 
```
*/15 * * * *   apache /usr/bin/scl enable rh-php71 -- php -f /usr/share/nextcloud/cron.php
```
works, on FreePBX we used solution @gsanchietti suggested because file is written by FreePBX and we just add those lines at the top of the file. (trust me, you don't want to know what happens down there)
Both should work